transforms/universal/noop/kfp_ray/noop_wf.py

```diff
@@ -23,22 +23,22 @@
 EXEC_SCRIPT_NAME: str = "noop_transform_ray.py"
 
 # components
-base_kfp_image = "quay.io/dataprep1/data-prep-kit/kfp-data-processing:latest"
+base_kfp_image = "quay.io/dataprep1/data-prep-kit/kfp-data-processing_v2:latest"
```

Comment:
No need to change the default image name

Reply:
Fixed.